### PR TITLE
Add example files for documentation

### PR DESCRIPTION
**Pull request by pierrepo**
Use example files for documentation directly from GitHub instead of Zenodo.
@HubLot could you review this PR?

### CODE REVIEW

**Comment by HubLot** [comment]
Why no more Zenodo ?

File: doc/source/PBcount.rst [hunk 1]
```diff
@@ -36,7 +37,7 @@ Example
 .. code-block:: bash
 
     $ PBcount -f psi_md_traj_1.PB.fasta -o psi_md_traj_1
-    read 90 sequences in psi_md_traj_1.PB.fasta
+    read 225 sequences in psi_md_traj_1.PB.fasta
```
Comment (HubLot) [inline doc/source/PBcount.rst]: It's not the same trajectory anymore ?
We have 225 frames now in psi_md_traj_1
This change also the output images

Comment (pierrepo) [inline doc/source/PBcount.rst]: This is the same base trajectory but since the demo data are not in the PyPI package anymore, I've added more snapshots.

**Comment by pierrepo** [comment]
No more Zenodo because I cannot add or remove easily files once a given dataset is defined.
But the full repository will be archived into Zenodo eventually.

File: doc/source/PBcount.rst (hunk 1 above)
Comment (HubLot) [inline doc/source/PBcount.rst]: Forget about the frames, the zenodo data have also 225 frames.
But I still don't know why the images changed.

Comment (HubLot) [inline doc/source/PBcount.rst]: Okay, we reply at the same time :)

Comment (pierrepo) [inline doc/source/PBcount.rst]: I changed images because I though I had too. A least, I am sure everything is up-to-date now ;-)

**Comment by pierrepo** [comment]
.*.npz files have been removed.

**Comment by HubLot** [comment]
There is still one :)

**Comment by pierrepo** [comment]
oups sorry :stuck_out_tongue_winking_eye:

**Comment by pierrepo** [comment]
@HubLot if it's OK for you, could you merge the PR please?

**Comment by pierrepo** [comment]
@HubLot can you merge?

**Pull request merged**